tests/lib/config/config-file.js

```diff
@@ -349,6 +365,32 @@ describe("ConfigFile", function() {
             assert.deepEqual(config, updatedConfig.eslintConfig);
         });
 
+        it("should load fresh information from a .eslintrc.js file", function() {
+            var initialConfig = {
```

Comment:
Comparison with `!==` works when requiring directly but `ConfigFile.load` always seems to return a new object:

```
it.only("should load fresh information from a .eslintrc.js file v2", function() {
    var config1 = ConfigFile.load(getFixturePath("js/.eslintrc.js"));
    var config2 = ConfigFile.load(getFixturePath("js/.eslintrc.js"));

    var config3 = require(getFixturePath("js/.eslintrc.js"));
    var config4 = require(getFixturePath("js/.eslintrc.js"));

    console.log( 'config1 !== config2', config1 !== config2 ); // --> true
    console.log( 'config3 !== config4', config3 !== config4 ); // --> false
    assert.notStrictEqual(config1, config2, 'via ConfigFile.load');
    assert.notStrictEqual(config3, config4, 'via require');
});
```